nitrite.go

```diff
@@ -460,3 +460,23 @@ func checkECDSASignature(
 
 	return ecdsa.Verify(publicKey, hashSigStruct, r, s)
 }
+
+func (d *Document) FromCosePayloadBytes(bytes []byte) error {
```

Comment:
Hmmm I think I was stuck in the json/cbor unmarshal mindset since that was my first approach but that's a good point. Updated usage to `NewDocumentFromCosePayloadBytes`